src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/AsymmetricAlgorithm.cs

```diff
@@ -9,6 +9,9 @@ namespace System.Security.Cryptography
 {
     public abstract class AsymmetricAlgorithm : IDisposable
     {
+        protected int KeySizeValue;
+        protected KeySizes[] LegalKeySizesValue;
```

Comment:
Do these need to be initialized to anything in the protected ctor?  Or they're expected to be 0/null in the derived instance?